mobile-widgets/qmlmanager.cpp

```diff
@@ -1938,17 +1938,19 @@ void QMLManager::showDownloadPage(QString deviceString)
 				.arg(connectionListModel.indexOf("Suunto"))
 				.arg(productList["Suunto"].indexOf("EON Core"))
 				.arg(connectionListModel.indexOf("USB device"));
-	} else if (deviceString.contains("mVendorId=11884") && // 0x2e6c / 0x3201
-		   deviceString.contains("mProductId=12801")) {
+	} else if (deviceString.contains("mVendorId=11884") && // 0x2e6c / 0x3201,0x3211,0x4201
+		   (deviceString.contains("mProductId=12801") ||
+		   (deviceString.contains("mProductId=12817") ||
+		   (deviceString.contains("mProductId=16897"))) {
```

Comment:
too much cut and paste... you don't need all of these leading opening parenthesis, only the first of the three... and this is currently causing a syntax error because you close one too few...

Reply:
Oops. Will fix.

Reply:
Ok, I've force-pushed the obvious fix, but still don't do a mobile build, so... 

Does github automatically redo things after the force-push?

Reply:
Yes it does :-)